Doc/reference/compound_stmts.rst

```diff
@@ -399,10 +399,10 @@ The execution of the :keyword:`with` statement with one "item" proceeds as follo
 #. The context expression (the expression given in the :token:`with_item`) is
    evaluated to obtain a context manager.
 
-#. The context manager's :meth:`__exit__` is loaded for later use.
-
 #. The context manager's :meth:`__enter__` method is invoked.
```

Comment:
The right sequence is:
* The `__enter__` method is loaded.
* The `__exit__` method is loaded.
* The `__enter__` method is invoked.

Reply:
@serhiy-storchaka You are right, I am going to update this, thanks.